robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAccessibilityEvent.java

```diff
@@ -121,7 +120,32 @@ public static boolean areThereUnrecycledEvents(boolean printUnrecycledEventsToSy
   public static void resetObtainedInstances() {
     obtainedInstances.clear();
     orderedInstances.clear();
-    sAllocationCount = 0;
+  }
+
+  /**
+   * Check for leaked objects that were {@code obtain}ed but never
+   * {@code recycle}d.
+   *
+   * @param printUnrecycledNodesToSystemErr - if true, stack traces of calls
+   *        to {@code obtain} that lack matching calls to {@code recycle} are
+   *        dumped to System.err.
+   * @return {@code true} if there are unrecycled nodes
+   */
+  public static boolean areThereUnrecycledNodes(boolean printUnrecycledNodesToSystemErr) {
```

Comment:
It has areThereUnrecycledEvents, areThereUnrecycledNodes was pasted here by mistake.